spec/unit/warehouse_spec.rb

```diff
@@ -4,19 +4,19 @@
   let(:empty_warehouse) { Warehouse.new(product_id: nil, amount: nil) }
   let(:warehouse) { Warehouse.new(product_id: 1, amount: 10) }
 
-  it "has 2 params" do
+  it "it has 2 params" do
```

Comment:
You don't need to write `it` inside test description. Because in full test description it will looks like `Warehouse it has 2 params` so don't write `it` in test description